worlds/bomb_rush_cyberfunk/Options.py

```diff
@@ -0,0 +1,161 @@
+from dataclasses import dataclass
+from Options import Choice, Toggle, DefaultOnToggle, Range, DeathLink, PerGameCommonOptions
+import typing
+
+if typing.TYPE_CHECKING:
+    from random import Random
+else:
+    Random = typing.Any
+
+
+class Logic(Choice):
+    """Choose the logic used by the randomizer."""
+    display_name = "Logic"
+    option_glitchless = 0
+    option_glitched = 1
+    default = 0
+
+
+class SkipIntro(DefaultOnToggle):
+    """Skips escaping the police station.
+    Graffiti spots tagged during the intro will not unlock items."""
+    display_name = "Skip Intro"
+
+
+class SkipDreams(Toggle):
+    """Skips the dream sequences at the end of each chapter."""
+    display_name = "Skip Dreams"
+
+
+class SkipHands(Toggle):
+    """Skips spraying the lion statue hands after the dream in Chapter 5."""
+    display_name = "Skip Statue Hands"
+
+
+class TotalRep(Range):
+    """Change the total amount of REP in your world.
+    At least 960 REP is needed to finish the game.
+    Will be rounded to the nearest number divisible by 8."""
+    display_name = "Total REP"
+    range_start = 1000
+    range_end = 2000
+    default = 1400
+
+    def round_to_nearest_step(self):
+        rem: int = self.value % 8
+        if rem >= 5:
+            self.value = self.value - rem + 8
+        else:
+            self.value = self.value - rem
+    
+    def get_rep_item_counts(self, random_source: Random, location_count: int) -> typing.List[int]:
+        def increment_item(item: int) -> int:
+            if item == 32:
+                item = 48
+            else:
+                item += 8
+            return item
+
+        items = [8]*location_count
+        while sum(items) < self.value:
+            index = random_source.randint(0, location_count-1)
+            while items[index] >= 48:
+                index = random_source.randint(0, location_count-1)
+            items[index] = increment_item(items[index])
+
+        while sum(items) > self.value:
+            index = random_source.randint(0, location_count-1)
+            if not (items[index] == 16 or items[index] == 24 or items[index] == 32):
+                index = random_source.randint(0, location_count-1)
+            items[index] -= 8
+
+        return [items.count(8), items.count(16), items.count(24), items.count(32), items.count(48)]
```

Comment:
`get_rep_item_counts` does not consistently give the correct number of items. On default settings, it sometimes gives 86 items instead of 87, leading to one less item being created than there are locations.